WordPress/src/test/java/org/wordpress/android/util/crashlogging/WPCrashLoggingDataProviderTest.kt

```diff
@@ -114,8 +118,14 @@ class WPCrashLoggingDataProviderTest : BaseUnitTest() {
     }
 
     @Test
-    fun `should provide empty application context`() = runTest {
-        assertThat(sut.applicationContextProvider.first()).isEmpty()
+    fun `should provide an application context of size 1`() = runTest {
+        assertThat(sut.applicationContextProvider.first().size).isEqualTo(1)
+    }
+
+    @Test
+    fun `should provide the webview version in the application context1`() = runTest {
```

Comment:
```suggestion
    fun `should provide the webview version in the application context`() = runTest {
```

🫣

Reply:
Good catch 😅 I committed you suggestion.